sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1059,9 +1059,12 @@ void
 handler::ext_oneapi_graph(command_graph<graph_state::executable>& graph)
 ----
 
-|Invokes the execution of a graph. Only one instance of `graph` may be executing,
-or pending execution, at any time. Concurrent graph execution can be achieved by
-finalizing a graph in modifiable state into multiple graphs in executable state.
+|Invokes the execution of a graph. Only one instance of `graph` will
+execute at any time. If `graph` is submitted multiple times, dependencies
+are automatically added by the runtime to prevent concurrent executions of 
+an identical graph. Note that synchronizations must be handled in the host 
+for level-zero backend, which may involve extra latency for subsequent 
```

Comment:
I'd move this last level zero specific sentence to the section at the end on "Non-implemented features and known issues"